pkg/repo/migrations/v2_3.go

```diff
@@ -9,8 +9,8 @@ import (
 
```

Comment:
we still need the previous config package, now known as `config_legacy` to perform this migration. When we are ready to remove this migration logic we can remove the `config_legacy` package.